### ISSUE TITLE
Bad titles

### ISSUE THREAD

**Issue by nknm** [ajayyy/DeArrow#255]
Some of the titles are just as bad if not worse than the original titles, I still get click baited there should be some clear guidelines to how to make a title, the thumbnails sometimes are also just as bad as the original ones but mostly they are quite good. But i find myself editing titles that were already edited before. 
![image](https://github.com/ajayyy/DeArrow/assets/22033631/dc0aaa5e-c16d-4e18-b88d-1ef0d1900153)
![image](https://github.com/ajayyy/DeArrow/assets/22033631/15679b5d-50f4-422f-9223-aa7e54d53a3c)

This video is 20% trying to give motivation to save TF2 and 50% explaining the lore of wow and 30% explaining what people did to save wow, that was make private servers, make petitions, until one person took it upon themselves to speak to the team (that person was a team member). The original title is straight up BS, this is not HOW we can save TF2 but how WoW was saved, the new title is better but there's no correlation between how to apply it to tf2 as wow got ruined via bad updates that could simply be reverted. and TF2 is a borderline dead game filled with bots and cheaters a petition wouldn't get many signatures nor we can revert players to how they were years ago. So i edited the title my self to "WoW saved their game with a petition for classic and persistence. Lets not give up on TF2" or another good title would be "When all hope was lost for WoW the players persistence saved it, lets do the same for TF2".
The only thing that can be applied from the WoW story to tf2 is really to have persistence and patience.

Another example here:

![image](https://github.com/ajayyy/DeArrow/assets/22033631/901ac614-5951-4b25-ab1b-fc2ccac79374)
![image](https://github.com/ajayyy/DeArrow/assets/22033631/1061157d-b8ff-4356-b3c3-29a6cb803427)

The video is literally about how the expiration date isnt what you think, not a basic guide on how to tell if food has expired. They explain in the video that the date means that its best by (once again good job with the thumbnial) and not that the food is unsafe to consume, then has a segment where they say that food should be given away after that date since stores can no longer sell it but people can still eat it, and at the end they say that u can spot if food is bad by smell, taste or how it looks. In my opinion this title is straight up wrong while the original was good. A better title would be "What expiration dates really mean and their evolution" or "How brands changed the label of "best by" to "expiration date" and how that came to be"

TLDR: I believe the titles are bad in quite a good amount of videos

**Comment by ajayyy** [comment]
We have guidelines see https://wiki.sponsor.ajay.app/w/DeArrow/Guidelines

**Comment by nknm** [comment]
Not for me, I assume someone has to accept the new titles or is there no supervision? I've read the guidelines and seems clear but like I said I feel like the the titles I mentioned there are not very good. But over all great app and great concept!
